docs/src/release-notes-java.md

```diff
@@ -18,10 +18,11 @@ The Network tab in the trace viewer has several nice improvements:
 
 ### Miscellaneous
 
-- The `mcr.microsoft.com/playwright-java:v1.47.0` now serves a Playwright image based on Ubuntu 24.04 Noble.
-  To use the 22.02 jammy-based image, please use `mcr.microsoft.com/playwright-java:v1.47.0-jammy` instead.
+- The `mcr.microsoft.com/playwright/java:v1.47.0` now serves a Playwright image based on Ubuntu 24.04 Noble.
+  To use the 22.02 jammy-based image, please use `mcr.microsoft.com/playwright/java:v1.47.0-jammy` instead.
+- The `:latest`/`:focal`/`:jammy` tag for Playwright Docker images is no longer being published. Pin to a specific version for better stability and reproducibility.
 - TLS client certificates can now be passed from memory by passing [`option: cert`] and [`option: key`] as byte arrays instead of file paths.
-- [`option: noWaitAfter`] in [`method: Locator.selectOption`] was deprecated.
+- [`option: setNoWaitAfter`] in [`method: Locator.selectOption`] was deprecated.
```

Comment:
Hmm... I was under impression that playwright.dev will automatically render options based on the language. If not, let's do that instead?

Reply:
Filed as https://github.com/microsoft/playwright/issues/32590.